tsunami/prompts/system.md

```
@@ -0,0 +1,1365 @@
+// Root component must be named "App"
+var App = app.DefineComponent("App", func(_ struct{}) any {
+	// UseLocal returns Atom[T] with Get() and Set() methods
+	todos := app.UseLocal([]Todo{
+		{Id: 1, Text: "Learn Tsunami", Completed: false},
+		{Id: 2, Text: "Build an app", Completed: false},
+	})
+	nextId := app.UseLocal(3)
+	inputText := app.UseLocal("")
+
+	// Event handlers
+	addTodo := func() {
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)